src/components/HornSelection.js

```diff
@@ -0,0 +1,34 @@
+
+import React, { Component } from 'react';
+import { Form, ListGroup } from 'react-bootstrap';
+
+class HornSelection extends Component {
+
+    handleChange = (event) => {
+    const selection = Number(event.target.value);
+    this.props.userSelection(selection);   
+    }
+
+    render() {
+        return (
+            <div>
+                <Form>
+                <Form.Select onChange={this.handleChange} aria-label="Default select example">
+                    <option>Open this select menu</option>
+                    <option value="1">One</option>
+                    <option value="2">Two</option>
+                    <option value="3">Three</option>
+                    <option value="100">One Hundred</option>
+                    <option value="-1">All</option>
+                    </Form.Select>
+            </Form>
```

Comment:
For this section, I think we can map() over this section and render different `<option>` tags with various values. Anytime we have repeating tags, we can maybe consider using map to render them dynamically. Depends on circumstance but it might work nicely here (you can put values in an array and perhaps do it this way).